index.html

```diff
@@ -19,10 +19,20 @@
                     <li class="nav-bar-li"><a href="#about-sec" class="nav-a">About</a></li>
                     <li class="nav-bar-li"><a href="#contact-sec" class="nav-a">Contact</a></li>
                     <li class="nav-bar-li"><a href="#contact-sec" class="nav-a"><img src="./images/mail_icon.png" alt="Mail icon"></a></li>
-                    <li id="mobile-icon"><a href="#"><img src="./images/Union.png" alt=""></a></li>
+                    <li id="mobile-icon" class="mob-icon is-active"><a href="#"><img src="./images/Union.png" alt=""></a></li>
+                    <li class="close-btn"><a href="#"><img src="./images/close_btn.png" style="height: 22px; width: 22px;" alt=""></a></li>
                 </ul>
             </nav>
         </header>
+        <div class="mobile-body">
+            <ul class="nav-ul">
+                <li><a href="#recent-sec" class="nav-a">Portfolio</a></li>
+                <li><a href="#about-sec" class="nav-a">About</a></li>
+                <li><a href="#contact-sec" class="nav-a">Contact</a></li>
+                <li><a href="#contact-sec" class="nav-a"><img src="./images/mail_icon.png" alt="Mail icon"></a></li>
```

Comment:
- [ ] In my opinion, you should remove this `mai icon` because it is not part of the original design, and as you may recall, one of the project requirements is that your design should be as close to the original as possible. 😊

## Your design 
![Screenshot from 2023-02-06 23-32-36](https://user-images.githubusercontent.com/89228743/217111079-84607a33-c3c8-4217-b6ef-366eb41cfec0.png)

### The original disgn

![Screenshot from 2023-02-06 23-33-38](https://user-images.githubusercontent.com/89228743/217111142-d01c0660-8e4d-4d88-88cc-2fb3e83c28b1.png)


- [ ] Last but not least, please use the same font as on the original design for your links, which is `font-family: 'Inter';`. Like this 👇️

```
.mobile-body .nav-ul li a {
    letter-spacing: 1px;
    font-weight: 700;
    font-size: 32px;
    line-height: 44px;
    color: white;
    font-family: 'Inter';
}
```